postgresql_audit/flask.py

```diff
@@ -6,17 +6,40 @@
 from flask import g, request
 from flask.globals import _app_ctx_stack, _request_ctx_stack
 
-from .base import VersioningManager
+from .base import VersioningManager as BaseVersioningManager
 
 
-class FlaskVersioningManager(VersioningManager):
-    @property
-    def transaction_values(self):
+class VersioningManager(BaseVersioningManager):
+    _actor_cls = 'User'
+
+    def get_transaction_values(self):
         values = copy(self.values)
-        if hasattr(g, 'activity_values'):
+        if context_available() and hasattr(g, 'activity_values'):
             values.update(g.activity_values)
+        if 'client_addr' not in values:
+            values['client_addr'] = self.default_client_addr()
+        if 'actor_id' not in values:
+            values['actor_id'] = self.default_actor_id()
         return values
 
+    def default_actor_id(self):
```

Comment:
I don't have a good alternative to this now that the tests are run without using the global `versioning_manager` instances. Is it okay that we just keep this in the PR? I'm feeling a bit exhausted by all the rebasing and refactoring 😩